Organize Sample Code (#2912)
nasa-gcn/gcn.nasa.gov: app/routes/docs.sample.text.md

---
handle:

>   breadcrumb: TEXT

Reviewer: ```suggestion
  breadcrumb: Plain Text
```

---

> # GCN Notices Emails

Author: ```suggestion
# Plain Text Messages
```

> # Plain Text Messages

Reviewer: ```suggestion
# Plain Text Format Notices
```

> # GCN Classic Text Messages

Author: ```suggestion
# GCN Notices Emails
```

>   breadcrumb: Plain Text Emails
> ---
> 
> # Plain Text Emails

Reviewer 2: ```suggestion
  breadcrumb: Plain Text
---

# Plain Text
```

> GCN Classic Notices are also distributed as email messages. To extract header information, one can use Python's built-in [`email`](https://docs.python.org/3/library/email.html) library.

Reviewer 2: ```suggestion
GCN Classic Notices are also distributed as plain text. To extract header information, one can use Python's built-in [`email`](https://docs.python.org/3/library/email.html) library.
```

> GCN Classic Notices are also distributed as email messages. To extract useful information from these alerts, one can use Python's built-in [`email`](https://docs.python.org/3/library/email.html) library.

Reviewer: ```suggestion
GCN Classic plain text-format Notices and GCN Circulars can be distributed as email messages. To extract useful information from these alerts, one can use Python's built-in [`email`](https://docs.python.org/3/library/email.html) library.
```
